doc/specs/#1012 - Show dependencies.md

```diff
@@ -0,0 +1,31 @@
+---
+author: Florencia Zanollo fzanollo/t-fzanollo@microsoft.com
+created on: 2021-05-28
+last updated: 2021-05-28
+issue id: 1012
+---
+
+# Show dependencies
+
+For [#1012](https://github.com/microsoft/winget-cli/issues/1012)
+
+## Abstract
+Several packages require other packages as dependencies. The Windows Package Manager should be able to support declared dependencies and inform the user about any required dependencies.

```

Comment:
I would suggest explicitly stating that this is the first step towards full support for dependencies to make it clear that we do not intend to stop at simply informing the user.